athena-elasticsearch/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
What does this file do? can we add some comments?

Reply:
This is a configuration file used by Mockito to enable support for final methods. I don't think I can add a comment inside, so I'll add a README file.